src/css/app.sass

```diff
@@ -221,7 +251,40 @@ body.body--dark
     padding: 4px !important
 
 @media only screen and (max-width: 550px)
+  .homeInfo
+    .text-h6
+      font-size: 1.3em
+  .q-tab__label
+    font-size: 11px
+  #connect-button button
```

Comment:
`ConnectButton` is a self-contained component, its styles should be defined in `ConnectButton.vue` -> `style`
if you do choose to move styles there, you can remove the `id` from `MainLayout.vue:36`

Reply:
sure but this was for re-usability, if we put it int there the responsive will apply to all instances of it

Reply:
This style is at the root level in app.sass, so this style is applied globally as written

Reply:
well yeah that topbar connect button is on every page. If we setup an additional button it won't have that responsive style as we won't give it the id.

Reply:
actually that button already has an id that is used within some js and hence can only be used once, so I moved the responsive there